docs/editors-guide/f-entities.md

```diff
@@ -193,6 +193,7 @@ MONDO:entailed  |   An inferred superclass (which is a redundant axiom) |   subc
 MONDO:equivalentObsolete    |   Used for cases where we have exact 1:1 matches between a live class in Mondo and an obsolete class in the source ontology. We want to avoid making an equivalence axiom (MONDO:equivalentTo) here.   |   xrefs   |   N   |   MONDO:0020499 'Nipah virus disease' database_cross_reference: Orphanet:1239 (was obsoleted in Orphanet) {source=MONDO:equivalentObsolete}
 MONDO:equivalentTo  |   This is interpreted strongly as an OWL equivalence axiom, meaning the two terms are considered to be exactly the same. This can be added to any xref where this is true.   |   xrefs   |   N   |   MONDO:0100087 'familial Alzheimer disease' database_cross_reference: GARD:0000632 (refers to the disease in GARD) {source=MONDO:equivalentTo}
 MONDO:includedEntryInOMIM | This indicates a term is an 'included' entry in OMIM, for example https://omim.org/entry/233910 | xrefs | N | MONDO:0100167 'dystonia, dopa-responsive, with or without hyperphenylalaninemia, autosomal recessive'
+MONDO:omimFormerly | Used on OMIM synonyms that have been deprecated in OMIM; in OMIM, these synonyms end with ", formerly". | synonyms | N | [OMIM:101000](https://omim.org/entry/101000?search=FORMERLY&highlight=formerly), Schwannomatosis, vestibular (Acoustic schwannomas, bilateral, formerly)
```

Comment:
@twhetzel Nico said I don't need to have `MONDO:omimFormerly` or `MONDO:omimIncluded` on the [entities](https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/) page, because he doesn't think they should be used as either values or annotation properties, but instead should be synonym types, and those don't belong here.

Context: ([1](https://docs.google.com/document/d/1zsfl5rkxdAhCk9xFX6oBDSa5UAjncBsloFuKU9Yt4Lo/edit?disco=AAABacqdWMQ)), ([2](https://github.com/monarch-initiative/mondo-ingest/issues/715#issuecomment-2546243282))

Gonna close. If Nico is mistaken, feel free to re-open!